run_dir/design/controls.html

```diff
@@ -0,0 +1,95 @@
+{% extends "base.html" %}
+<!--
+Template file: controls.html
+URL: /controls/
+Title: Controls
+Description: Overview of positive and negative controls
+-->
+
+{% block stuff %}
+<h1 class="worksets_page_heading"><span id="page_title">Positive and Negative Controls</span></h1>
+</div>
+  <div id="page_content" >
+    <div class="tabbable">
+      <ul class="nav nav-tabs" role="tablist">
+        <li class="nav-item"><a class="nav-link active" href="#tab_negative_controls" role="tab" data-toggle="tab" aria-selected="true">Negative Controls</a></li>
+        <li class="nav-item"><a class="nav-link" href="#tab_positive_controls" role="tab" data-toggle="tab">Positive Controls</a></li>
+      </ul>
+      <div class="tab-content">
+        <!-- loop through the two control types -->
+        {% for control_type in ["negative", "positive"] %} 
+          <!-- adding data to specific tabs -->
+          <div class="tab-pane fade show active" id="tab_{{ control_type }}_controls"> 
+            <h2 class="{{ control_type }}_controls_heading"><small class="text-muted">All {{ control_type }} controls with initiated worksets</small></h2>
+            <div id="{{ control_type }}_control-list">
+              <!-- creating the data table -->
+              <table class="table table-striped table-bordered sortable" id="{{ control_type }}_controls_table">
+                <thead id="{{ control_type }}_controls_table_head">
+                  <tr class="sticky darkth">
+                  {% for onekey in headers %}
+                      <th class="sort">{{ onekey[0] }}</th>
+                  {% end %}
+                  </tr>
+                </thead>
+                <!-- footer content to be replaced by search boxes -->
+                <tfoot id="{{ control_type }}_controls_table_footer">
+                  <tr class="darkth">
+                    {% for onekey in headers %}
+                      <th class="sort">{{ onekey[0] }}</th>
+                    {% end %}
+                  </tr>
+                </tfoot>
+                <!-- filling the data table with control sample info -->
+                <tbody id="{{ control_type }}_controls_table_body">
+                  {% for control in all_control_data[control_type]%}
+                  <tr>
+                    {% for onekey in headers %}
+                      {% if onekey[1] == 'project' %}
```

Comment:
What is the difference between the cases `project`, `customer_name`, `status_manual`?

Reply:
project is the Project ID (for example P31310), customer_name is the given sample name (for example Amplicon_Neg_Control_2) and status_manual is the status as set in the LIMS project overview. Very different things